test/sqllogictest/role_create.slt

```diff
@@ -63,7 +63,7 @@ COMPLETE 0
 simple conn=regress_role_limited_admin,user=regress_role_limited_admin
 CREATE ROLE regress_nosuch_superuser SUPERUSER;
 ----
-db error: ERROR: SUPERUSER attribute is not supported, for more information consult the documentation at https://materialize.com/docs/sql/create-role/#details
+db error: ERROR: self-managed auth is not enabled, setting password or superuser is not allowed are not supported
```

Comment:
This doesn't seem right, ideally for a cloud customer we shouldn't mention anything about self-managed :)